src/opm/parser/eclipse/EclipseState/Schedule/UDQ/UDQDefine.cpp

```diff
@@ -339,7 +340,9 @@ std::pair<UDQUpdate, std::size_t> UDQDefine::status() const {
     return std::make_pair(this->m_update_status, this->m_report_step);
 }
 
-
+std::vector<UDQToken> UDQDefine::tokens() const {
```

Comment:
I guess this could return a `const std::vector<UDQToken>&`

Reply:
Agree -  OK.